src/components/Header/Header.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import './Header.css'
+import rottenFruit from "../../assets/rottenFruit.png"
```

Comment:
I am so glad this is working now, went down a pretty deep rabbit hole trying to figure out why this wasn't working, great work!